tests/test_json_schema.py

```diff
@@ -1741,7 +1741,7 @@ class Model(BaseModel):
         something: Optional[int] = None
 
         @validator('something')
-        def check_field(cls, v, *, values, config, field):
+        def check_field(cls, v, *, values, config, context):
```

Comment:
Is it possible to get the field that is being validated in V2? Does it come in `context`?